runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -8140,6 +8249,16 @@ const char* am_reqStr(c_nodeid_t tgtNode, amRequest_t* req, size_t reqSize) {
     }
     break;
 
+  case am_opFAMOResult:
+    len += snprintf(buf + len, sizeof(buf) - len,
```

Comment:
D'oh! I looked above the switch-stmt but not below. Okay, that's good then.

(I hadn't even thought about the fact that using the `am_opFAMOResult` AM's `b.pAmDone` field to carry the original `am_opAMO` initiator's 'done' flag address means that `am_opFAMOResult` AMs _have_ to be non-blocking. I.e., we couldn't do them as blocking AMs even if we wanted to, because then we would need two `pAmDone` fields. But this is just a curiosity, because I can't imagine we'll ever want to do them as blocking.)